company-styles.css

```diff
@@ -2,32 +2,107 @@ body {
     background-color: #f1e8e9;
 }
 
-nav {
-    font-size: 5mm;
+.nav-flexbox {
+    font-size: 7mm;
     background-color: #8a9ea7;
-    border: 20px solid #8a9ea7 ;
-    padding: 0 5%;
+    /* border: 20px solid #8a9ea7 ; */
```

Comment:
Good idea to remove comments when you are ready for review :smile:

Reply:
updated and sent push to repo